src/schema/v2/createPartnerOfferMutation.ts

```diff
@@ -57,22 +58,33 @@ export const createPartnerOfferMutation = mutationWithClientMutationId<
     discount_percentage: { type: GraphQLInt },
   },
   outputFields: {
+    partner: {
+      type: PartnerType,
+      resolve: (result) => result.partner || null,
+    },
     partnerOfferOrError: {
       type: ResponseOrErrorType,
       description: "On success: the partner offer created.",
-      resolve: (result) => result,
+      resolve: (result) => result.partnerOffer || result,
```

Comment:
Just `result => result` or `result => result.partnerOffer`, and we can update the code above in the success / fail type to accomodate. `result.partnerOffer` is structurally different than `result` because result returns an object with two sub-properties; without being clear here, this will likely result in a future bug.